engines/aws/app/models/aws/cli.rb

```diff
@@ -64,15 +74,9 @@ def create_vpc(cidr_block='192.168.0.0/16', name='vpc')
 
     def describe_vpc(vpc_id)
       args = %W(ec2 describe-vpcs --vpc-ids #{vpc_id})
-      begin
-        stdout, stderr = execute(*args)
-        return stderr if stderr.present?
-        return stdout
-      rescue Cheetah::ExecutionFailed => err
-        if err.stderr.include?('InvalidVpcID.NotFound')
-          "{\"Vpcs\": [{\"State\": \"not_found\"}]}"
-        end
-      end
+      get_description(
+        args, 'InvalidVpcID.NotFound', "{\"Vpcs\": [{\"State\": \"not_found\"}]}"
```

Comment:
I know I did it that way, but it was dumb then and it's still dumb. Please use single-quotes for the not_found JSON snippet, so you don't need to escape the double-quotes. E.g. `'{"Vpcs": [{"State": "not_found"}]}'